File: 1 Periodo/Trabalhos/Trabalho 2/soma.jl
#=
Linguagem: Julia
•SomaNumero - Programa que lê dois números do usuário e imprima a soma desses números na tela.
=#

println("Escolha dois valores para serem somados:")
println("Valor 1:")
numero1 = readline()
numero1 = parse(Int64, numero1)
println("Valor 2:")
numero2 = readline()
numero2 = parse(Int64, numero2)

soma = numero1 + numero2

println("O valor da soma entre ", numero1," e ", numero2, " é igual a ", soma)

File: 1 Periodo/Laboratorios/Laboratorio 2/Procedural/soma.lua
--[[
Linguagem: Lua
Paradigma: Procedural
•SomaNumero - Programa que lê dois números do usuário e imprima a soma desses números na tela.
--]]

print("Escolha um valor para o 1º número: ")
local numero1 = io.read("*f") -- leitura da string com o io.read(), leitura de um número flutuante com o especificador "*f"
print("Escolha um valor para o 1º número: ")
local numero2 = io.read("*f")

local soma = numero1 + numero2

print(string.format("O valor da soma entre %g e %g resulta em %g", numero1, numero2, soma))
